dev/versioning.md

```diff
@@ -48,3 +48,11 @@ A major release is likely to break your lint build.
 
 * Adding new rules in a config set
 * Making a rule more restrictive
+
+## Exemptions
+
+There are some files that are intended to be used for special purposes and as such are exempt from this versioning policy.
+
+Exemption list:
+
+* lib/config/a11y.js
```

Comment:
Let's refer to this in the way that users would (I doubt most folks know that specifying `recommended` maps to this repos `lib/config/recommended.js`). 

```suggestion
* `a11y`
```

Reply:
Can you also add a small description of _why_ this config is exempted?